lib/workers/repository/update/branch/handle-existing.ts

```diff
@@ -44,3 +51,50 @@ export async function handlepr(config: BranchConfig, pr: Pr): Promise<void> {
     logger.debug(`Merged PR with PrNo: ${pr.number} is blocking this branch`);
   }
 }
+
+export async function handleModifiedPr(
+  config: BranchConfig,
+  pr: Pr
+): Promise<void> {
+  if (config.suppressNotifications!.includes('prEditedNotification')) {
+    return;
+  }
+
+  const editedPrCommentTopic = 'Edited/Blocked Notification';
+  const content =
+    'Renovate will not automatically rebase this PR, because it does not recognize the last commit author and assumes somebody else may have edited the PR.\n' +
+    'You can manually request rebase by checking the rebase/retry box above.\n\n' +
+    emojify(' :warning: **Warning**: custom changes will be lost.');
+
+  const dependencyDashboardCheck =
+    config.dependencyDashboardChecks?.[config.branchName];
```

Comment:
Why not passing the computed value from calling function?

Reply:
We could do that for sure, but than we will have 3 function parameters:

```ts
export async function handleModifiedPr(
  config: BranchConfig,
  pr: Pr,
  rebaseRequested: boolean
): Promise<void> {...}
```
config is needed for `config.suppressNotifications` regardless of the third arg. and the third arg can be calculated using the already passed branch config. so i guess having the third arg would result in a duplicate data (?).

I also feel that `config.suppressNotifications` should be handled inside this function and not in the calling function.